System/DataModule.cpp

```diff
@@ -1,764 +1,497 @@
-//////////////////////////////////////////////////////////////////////////////////////////
-// File:            DataModule.cpp
-//////////////////////////////////////////////////////////////////////////////////////////
-// Description:     Source file for the DataModule class.
-// Project:         Retro Terrain Engine
-// Author(s):       Daniel Tabar
-//                  data@datarealms.com
-//                  http://www.datarealms.com
+#include "DataModule.h"
+#include "RTEManagers.h"
 
+namespace RTE {
 
-//////////////////////////////////////////////////////////////////////////////////////////
-// Inclusions of header files
+	const std::string DataModule::m_ClassName = "DataModule";
+
+/////////////////////////////////////////////////////////////////////////////////////////////////////////////////////////////
+
+	void DataModule::Clear() {
```

Comment:
I will comment on DataModule.cpp separately, time has defeated me for now but I want to get this review up for you.